Scripts/Mac/BuildInstaller.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+. "$(dirname ${BASH_SOURCE[0]})/InitializeEnvironment.sh"
+
+CONFIGURATION=$1
+if [ -z $CONFIGURATION ]; then
+  CONFIGURATION=Debug
+fi
+
+PACKAGEVERSION=$2
+if [ -z $PACKAGEVERSION ]; then
+  PACKAGEVERSION="0.0.0.1"
+fi
+
+INSTALLROOT=$VFS_OUTPUTDIR/"InstallerRoot"
```

Comment:
There's no need for "Root" in the name of the folder on disk. Please follow the pattern used by the other projects, which is that the `BuildOutput` folder's children are named for the project that built them, and also have their own subfolders for platform and configuration. On Windows, the installer ends up in `$VFS_OUTPUTDIR/GVFS.Installer/bin/x64/$CONFIGURATION`. Using that pattern will also enable us to integrate this installer into our `GVFS.sln` with a csproj that calls out to this script. That will enable the IDE build to create the installer, and it will also give us incremental builds "for free" if we set up the csproj correctly.

Reply:
Have created a new installer csproj for Mac, which invokes the CreateMacInstaller.sh script during build. Please take a look.